src/test/resources/TestNG.xml

```diff
@@ -7,6 +7,11 @@
         <classes>
             <class name="LoginTests"/>
             <class name="BaseTest"/>
+<<<<<<< Updated upstream
+=======
+            <class name="Homework16"/>
+            <class name="Homework17"/>
+>>>>>>> Stashed changes
         </classes>
     </test>
 </suite>
```

Comment:
You got this problem, please correct it. 

![image](https://github.com/Test-Pro-Bootcamp/A47-QA-Automation-Framework/assets/104761482/e5947771-ad24-47b4-87fa-5aa68c848273)

HINT

![image](https://github.com/Test-Pro-Bootcamp/A47-QA-Automation-Framework/assets/104761482/05d88f5b-bfb0-40a2-90cb-e2307a18846d)